packages/dds/tree/src/shared-tree/treeAlpha.ts

```diff
@@ -314,6 +314,8 @@ export const TreeAlpha: TreeAlpha = {
 		} & ICodecOptions,
 	): Unhydrated<TreeFieldFromImplicitField<TSchema>> {
 		const content: ViewContent = {
+			// TODO: Either the return type from this method cannot be persisted due to type constraints, or we need to
```

Comment:
This value is consumed on line 324, which is going to parse it using the exact save version of the code currently running: no persistence actually occurs here. https://github.com/microsoft/FluidFramework/pull/24562 addresses this case